ui/components/app/token-list-display/token-list-display.scss

```diff
@@ -0,0 +1,43 @@
+.loading-span {
+    display: flex;
+    height: 250px;
+    align-items: center;
+    justify-content: center;
+    padding: 30px;
+}
+
+.token-list-item {
+    padding: 8px;
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    cursor: pointer;
+    padding: 8px;
+
+    &:hover {
+        background-color: rgba($alto, 0.2);
+    }
+
+    &__data {
+        margin-left: 8px;
+    }
+
+    &__symbol {
+        font-size: 1rem;
+        line-height: 140%;
+        margin-bottom: 2px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+    }
+
+    &__balance {
+        font-size: 0.75rem;
```

Comment:
[REQ] Please use the Typography mixins H1-H9,and Paragraph instead of setting font-size manually. There's also the Typography component that can be used to avoid some commonly needed styles.

Reply:
Special note: you may not be able to use the full mixin for the symbol (due to setting font family etc) but there is also a font-size variable that you can use instead of the literal rem value